### PR TITLE
Events page updated

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
As per the new design mock up event page design updated

### CODE REVIEW

**Comment by duttakapil** [comment]
@M-ZubairAhmed is it possible for you to host this somewhere and share a demo link, so that people can interact with the what you've created?

**Comment by M-ZubairAhmed** [comment]
@duttakapil we have mentioned how to run the dev server in our readme i guess developers should be able to run a local copy. 
If you want we can have animated gifs here!

File: package.json [hunk 1]
```diff
@@ -16,19 +16,28 @@
...
     "space:": 2,
-    "ignores": ["next.config.js"]
+    "ignores": [
+      "next.config.js"
+    ]
   },
   "lint-staged": {
     "*.js": [
-      "prettier --write --single-quote --print-width=80 --trailing-comma=all",
+      "prettier --write --single-quote --print-width=80 --trailing-comma=all --no-semi",
```
Comment (vinaypuppal) [inline package.json]: Why did you disable semicolons?

Comment (M-ZubairAhmed) [reply package.json]: more of a coding style and also one less thing to worry about

File: common/urls.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const pastEventsMeetupURL =
```
Comment (vinaypuppal) [inline common/urls.js]: Move this file under `utils` folder

File: components/row-events.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  return (
+    <cardhead onClick={() => window.open(props.link)}>
+      {/* <div style={{ display: 'inline-block' }}>Picture goes here</div> */}
+      <Card
+        style={{
+          marginTop: '15px',
+          width: '800px',
+          backgroundColor: '#fafafa',
+        }}
+      >
```
Comment (vinaypuppal) [inline components/row-events.js]: Please move all inline styles to `<style jsx>` tag

File: package.json (hunk 1 above)
Comment (vinaypuppal) [reply package.json]: If you don't like semicolons then you don't type them in code, let tools like prettier and xo place them at the time of committing(which will be taken care by pre-commit hook `lint-staged`).

File: pages/events.js [hunk 4]
```diff
@@ -1,24 +1,108 @@
...
+      NProgress.start()
+      try {
+        const requestPastEvents = await axios.get(pastEventsMeetupURL)
+        NProgress.set(0.4)
+        const requestFutureEvents = await axios.get(futureEventsMeetupURL)
+        await this.setState({
+          pastEvents: requestPastEvents.data,
+          futureEvents: requestFutureEvents.data,
+        })
+        NProgress.done()
```
Comment (vinaypuppal) [inline pages/events.js]: Please move this logic to `getInitialProps` so page can be Server Side Rendered. More Info https://github.com/zeit/next.js/#fetching-data-and-component-lifecycle

File: pages/events.js [hunk 5]
```diff
@@ -1,24 +1,108 @@
...
+      return (
+        <div>
+          <div
+            style={{
+              backgroundColor: '#f4f7fb',
+              minHeight: '200px',
+            }}
+          >
+            <h1
+              style={{
```
Comment (vinaypuppal) [inline pages/events.js]: Move inline styles to `<style jsx>`

**Comment by vinaypuppal** [comment]
@M-ZubairAhmed It has lot of unrelated changes would you revert them back and submit a new PR with just event page updates.

BTW if you don't like semicolons then you don't type them in code, let tools like prettier and xo place them at the time of committing(which will be taken care by pre-commit hook `lint-staged`).

**Comment by vinaypuppal** [comment]
This is the preview url https://coderplex-app-assdaiebwd.now.sh/events
@M-ZubairAhmed Next time use now.sh to deploy your changes and place url in your PR
@duttakapil We can automate this process if we have a paid now.sh account because OSS plan allows only 3 instances to be active at a point of time.

**Comment by vinaypuppal** [comment]
@M-ZubairAhmed There seems some CORS issue in above url page is not loading events 
screen shot 2017-10-16 at 19 28 15

**Comment by M-ZubairAhmed** [comment]
yes, i raised the issue in coderplex-backend

**Comment by vinaypuppal** [comment]
@M-ZubairAhmed Its not related to our backend, its from meetup API

File: .gitignore [hunk 6]
```diff
@@ -116,4 +116,7 @@ $RECYCLE.BIN/
 
 # Others
 data/
-.next/
\ No newline at end of file
+.next/
+
+# generate lock files
+package-lock.json
```
Comment (vinaypuppal) [inline .gitignore]: `package-lock.json` should not be ignored. And we should both either use either `yarn` or `npm` only not both. As this repo already has `yarn.lock` file so you should continue with it.

**Comment by M-ZubairAhmed** [comment]
Demo link 
https://coderplex-app-fjnfsvpvig.now.sh/events

**Pull request merged**